ui/admin/app/components/form/field/list-wrapper/text-input/index.hbs

```diff
@@ -16,7 +15,7 @@
               @value={{option.value}}
               @type='text'
               disabled={{@disabled}}
-              aria-labelledby={{valueHeaderID}}
+              aria-label={{valueHeaderID}}
```

Comment:
Note: since these fields don't have a visible label, we should use `aria-label` for accessibility conformance. `aria-labelled by` is used when we have visible label fields like the key-value component

Reply:
great catch!!

Reply:
We shouldn't be using a GUID then. The label should be human readable otherwise we would be reading back a GUID to the user.

We were treating the table header as a "label" which was human readable and linked it with the ID and `aria-labeledby`, if we're just using `aria-label`, we need to just give it real text.

You can confirm this with your screen reader, give it a try. We should get more into the habit of checking with a screen reader